gbsplay.c

```diff
@@ -267,6 +267,7 @@ int main(int argc, char **argv)
 
 	/* clean up terminal */
 	restore_terminal();
+	printf("\033[?25h");
```

Comment:
This breaks all plugouts that write to stdout because the verbosity setting is not honored.
On my system this also made the tests fail immediately on the first call to "make" for this branch.

Are you developing on MINGW32 or MINGW64 where configure disables the tests automatically?